app/routes/wallet/dids.py

```diff
@@ -83,12 +116,27 @@ async def get_public_did(
     return result.result
 
 
-@router.put("/public", response_model=DID)
+@router.put("/public", response_model=DID, summary="Set Public DID")
 async def set_public_did(
     did: str,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> DID:
-    """Set the current public DID."""
+    """
+    Set the current public DID
+    ---
+
+    This endpoint allows you to set the current public DID.
+    The tenant needs a connection with the endorser to make a did public.
```

Comment:
Let's say "with an endorser", and include a note somewhere (by default, only issuers have a connection with the endorser, and so only issuers can update their public did)